connector_prestashop/models/product_template/common.py

```diff
@@ -166,7 +168,20 @@ def export_quantity(self, filters, quantity):
     def export_quantity_url(self, filters, quantity, client=None):
         if client is None:
             client = self.client
+        # Get the ids of the 'stock_availables' records for this product.
+        # Thereafter, we will update those records with the new quantities.
         response = client.search(self._prestashop_model, filters)
+        if not response:
```

Comment:
Can it be that there's response, but with empty list as stated in the main comment?

Reply:
Here `responses` is the list of ids, `search` returning a list of ids, the variable name is a bit confusing in that regard.

Reply:
Well, for sure we will ask the same in the future, so if you don't mind, it would be best, but I won't block it for that reason. A version number changed would also be desirable, but as there are other PRs from @simahawk, I don't want to be again a conflictive change, so I will merge all together so that an effective version change happens.

Reply:
Agreed, it's renamed. As for the version number, yeah... the conflict would be an issue :-(
Thanks!